inluxclient/query.go

```diff
@@ -0,0 +1,315 @@
+package influxclient
+
+import (
+	"encoding/base64"
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"io"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// TableColumn holds flux query table column properties
+type TableColumn struct {
+	Name         string
+	DataType     string
+	IsGroup      bool
+	DefaultValue string
+}
+
+// String returns TableColumn string dump
+func (f *TableColumn) String() string {
```

Comment:
I don't think this method pulls its weight - `Printf("%#v", col)` would produce something very similar.